source/_components/binary_sensor.command_line.markdown

```diff
@@ -23,12 +23,12 @@ To use your Command binary sensor in your installation, add the following to you
 # Example configuration.yaml entry
 binary_sensor:
   - platform: command_line
-    command: cat /proc/sys/net/ipv4/ip_forward
+    command: 'cat /proc/sys/net/ipv4/ip_forward'
 ```
 
 {% configuration %}
 command:
-  description: The action to take to get the value.
+  description: The action to take to get the value. It's highly recommended to enclose the command in single quotes `'` as it ensures all characters can be used in the command and reduces the risk of unintentional escaping. To include a single quote in a command enclosed in single quotes, double it: `''`.
```

Comment:
```suggestion
  description: The action to take to get the value. It's highly recommended to enclose the command in single quotes `'` as it ensures all characters can be used in the command and reduces the risk of unintentional escaping. To include a single quote in a command enclosed in single quotes, double it: `''`.
```

Will solve the build error or remove the `:` after "double it"

Reply:
@klaasnicolaas Your suggested change has no change, can you try again?

Reply:
What do you mean with try again?

Reply:
I mean, your suggested change is same as what is already there. Maybe github is being buggy.. Can you explain how I should change to solve the build error again?

Reply:
nvm, moved the warning as you suggest below